openstack/identity/v3/projects/requests.go

```diff
@@ -153,6 +168,9 @@ type UpdateOpts struct {
 
 	// Description is the description of the project.
 	Description *string `json:"description,omitempty"`
+
+	// Tags is a list of tags to associate with the project.
+	Tags []string `json:"tags,omitempty"`
```

Comment:
How can someone remove _all_ tags from a project? Should this be `*[]string`?

Reply:
Just reviewed other resources; they tend to use pointers when updating tags. So I aligned project to those other resources and implemented the change in a new commit.